lib/services/local/presence.go

```diff
@@ -52,10 +52,10 @@ type backendItemToResourceFunc func(item backend.Item) (types.ResourceWithLabels
 
 // NewPresenceService returns new presence service instance
 func NewPresenceService(b backend.Backend) *PresenceService {
-	return NewPresenceServiceV2(b, nil)
+	return NewIdemeumPresenceService(b, nil)
```

Comment:
do you want to call it IdemeumNewPresenceService?

Reply:
New* is convention they follow to create a new object, so i have followed the same.